lang/lang_en_fr.txt

```diff
@@ -946,51 +938,51 @@
 "Resuming print"
 "Reprise de l'impr."
 
-#MSG_BED_CORRECTION_RIGHT c=14 r=1
+#MSG_BED_CORRECTION_RIGHT c=14
 "Right side[um]"
 "Droite [um]"
 
-#MSG_RPI_PORT
+#MSG_RPI_PORT c=13
 "RPi port"
 "Port RPi"
 
 #MSG_WIZARD_RERUN c=20 r=7
 "Running Wizard will delete current calibration results and start from the beginning. Continue?"
 "Lancement de l'Assistant supprimera les resultats actuels de calibration et commencera du debut. Continuer?"
 
-#MSG_SD_CARD
+#MSG_SD_CARD c=8
 "SD card"
 "Carte SD"
 
-#MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY
+#MSG_TOSHIBA_FLASH_AIR_COMPATIBILITY c=8
 "FlashAir"
 "FlshAir"
```

Comment:
```suggestion
"FlashAir"
```
Is there a reason that this is translated to `FlshAir`? All the documentation I can find on FlashAir in french calls the card `FlashAir`.

Reply:
Just changed it to use the origin message